Packages/com.unity.inputsystem/InputSystem/Utilities/NumberHelpers.cs

```diff
@@ -13,6 +13,15 @@ public static int AlignToMultipleOf(this int number, int alignment)
             return number + alignment - remainder;
         }
 
+        public static long AlignToMultipleOf(this long number, long alignment)
```

Comment:
Should we add a test just for this function?

Reply:
I'll go add something in the next PRs.